odkx-src/sync-endpoint-user-instructions.rst

```diff
@@ -51,6 +51,11 @@ Assigning users to groups
   3. This list is all the groups under *ou=default_prefix*.
   4. Click on the group that you want to assign users to.
   5. A few groups are created when the LDAP server is brought up, refer to :doc:`data-permission-filters` for descriptions of these groups.
+
+  .. note::
+    
+     A user needs to be assigned one of the roles in addition to any other group of your choosing. These roles are available as groups 500, 501, 502 503.
```

Comment:
I am a little confused about "any other group" here. I am also not sure what is meant by "These roles" and how you would know what role group 500 was, for example.

Reply:
Hello, Caroline! Thank you for pointing this out. I'll make another push that explains better.